### DISCUSSION TITLE
Start value has mixed support, consider using flex-start instead. Warn messsage from [vite:css]

### DISCUSSION THREAD

**Discussion by jesspinklet** [tailwindlabs/tailwindcss#11802] [Help]
![image](https://github.com/tailwindlabs/tailwindcss/assets/77070946/4ce0f643-0805-46aa-8167-384fdb41a074)

I'm using @nuxtjs/tailwindcss v6.8.0 but I don't think it's because of that. this warning is new, and I don't have any other styles other than imported ones for tailwind: base, component, utilities.

**Comment by hodenortypro** [comment]
I understand that you are using @nuxtjs/tailwindcss v6.8.0 and you don't think the warning is related to that. 
Since the warning is new and you only have imported styles for tailwind (base, component, utilities), it's possible that the warning is coming from another source. 
Have you made any recent changes to your code or added any new dependencies?

[Is Pickleball a professional sport](https://www.ballsportsgames.com/is-pickleball-a-professional-sport/)
